frontend/packages/console-shared/src/components/shortcuts/Shortcut.tsx

```diff
@@ -34,40 +37,43 @@ const Shortcut: React.FC<ShortcutProps> = ({
   keyName,
   rightClick,
   shift,
+  toggleTabAction,
 }) => {
   const { t } = useTranslation();
-  const isMac = window.navigator.platform.includes('Mac');
   return (
     <tr>
       <td className="ocs-shortcut__cell">
-        {(ctrl || (!isMac && ctrlCmd)) && <Command>Ctrl</Command>}
-        {alt && <Command>{isMac ? '⌥ Opt' : 'Alt'}</Command>}
-        {shift && <Command>Shift</Command>}
-        {isMac && ctrlCmd && <Command>⌘ Cmd</Command>}
+        {(ctrl || (!isMac && ctrlCmd)) && <ShortcutCommand>Ctrl</ShortcutCommand>}
+        {toggleTabAction && isMac && <ShortcutCommand>Shift</ShortcutCommand>}
```

Comment:
I wrote this same comment last time but I suppose I forgot to commit it.

Please remove `toggleTabAction` because it is too specific to your use case and doesn't belong in the `Shortcut` component. I provided an alternate solution in another comment.

Reply:
removed